app/controllers/concerns/two_factor_authenticatable_methods.rb

```diff
@@ -305,15 +301,14 @@ def unconfirmed_phone?
   end
 
   def phone_view_data
-    { confirmation_for_add_phone: confirmation_for_add_phone?,
+    {
+      confirmation_for_add_phone: confirmation_for_add_phone?,
       phone_number: display_phone_to_deliver_to,
       code_value: direct_otp_code,
       otp_expiration: otp_expiration,
       otp_delivery_preference: two_factor_authentication_method,
       otp_make_default_number: selected_otp_make_default_number,
-      voice_otp_delivery_unsupported: voice_otp_delivery_unsupported?,
-      unconfirmed_phone: unconfirmed_phone?,
-      account_reset_token: account_reset_token }.merge(generic_data)
```

Comment:
* `unconfirmed_phone` was still used in `PhoneDeliveryPresenter` ([source](https://github.com/18F/identity-idp/blob/04079b9d0631b5561efef128f9f4d6b696aa5b3d/app/presenters/two_factor_auth_code/phone_delivery_presenter.rb#L92))
* `voice_otp_delivery_unsupported?` and `account_reset_token` methods appear to be unused in this class now